nignanthomas/wayfarer
# #167889902 User should be able to book a trip

#### What does this PR do?
User should be able to book a seat on a trip

#### Description of Task to be completed?
Create SQL query in bookModel to get create a booking object in the database
Create tests for bookig a seat on trip feature

#### How should this be manually tested?
Clone the repo to your machine and run:
* `npm install`.
* `npm run dev`.
* create a postgres database named `dbname`.
* Download Postman.
* In Postman, create a new POST request tab and enter the following URL: `http://localhost:3000/api/v2/trips`.
After registration, create this object in Body raw/json:
{
	"trip_id": 7,
	"seat_number": 12
}
* Click `Send`

* You should have s similar response to the following:  
{
    "status": 201,
    "message": "Booking Created Successfully",
    "data": {
        "id": 6,
        "bus_license_number": "KLK 4694",
        "trip_date": "10-12-2020",
        "first_name": "John",
        "last_name": "Doe",
        "user_email": "johndoe@gmail.com",
        "seat_number": 12
    }
}

#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?
[#167889902 ](https://www.pivotaltracker.com/story/show/167889902 )

#### Screenshots (if appropriate)
![book-trip](https://user-images.githubusercontent.com/42959540/63017227-12207680-be96-11e9-8347-7b8c906272f4.PNG)
